.github/workflows/ci.yml

```
@@ -0,0 +1,70 @@
+name: Build macOS, Ubuntu, Windows
+
+on:
+  pull_request:
+
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: Release ${{ matrix.target }}
```

Nit, since this is for CI and not for release (CD) anymore:

```suggestion
    name: Target ${{ matrix.target }}
```